src/components/HomeListView.js

```diff
@@ -33,7 +29,7 @@ export class HomeListView extends Component {
         cellHeight={50}
         cellProps={{ onPress: this.onGroupSelect }}
         pageSize={5}
-        sectionHeader={() => null}
+        sectionHeader={nullFn}
```

Comment:
Just tried removing the null function, I was wrong, the AlphabetListView actually gives a default component which doesn't align correctly with the rest of the other components. If we want a SectionHeader component, we should create it in another PR.